### PR TITLE
Issue #492 - MacOS man

### LINKED ISSUE #492: MacOS man not working
I believe its not added to the install.sh script

### PR DESCRIPTION
**Pull request by organom**
MacOS man by default is stored in `/usr/local/shared/man/man1`  instead  of `/usr/local/man/man1`

### CODE REVIEW

**Comment by organom** [comment]
@ilyash  can you please test on both your mac and linux in order to make sure everything works correctly? thanks :)

**Comment by ilyash** [comment]
> @ilyash can you please also try the install both in your mac and linux in order to make sure everything works correctly? thanks :)

As agreed during the weekly, please add tests so that correct installation is tested during/after the build on GitHub

**Review by ilyash — CHANGES_REQUESTED** [review]
Please add tests

**Comment by organom** [comment]
@ilyash-b added the tests directly in the makefile at https://github.com/ngs-lang/ngs/pull/503/commits/b40e7988d6d704f539b897f788591a3ef7388403. Do you have any better idea, or is it ok like this?

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,7 @@ endif
 .PHONY: tests
 tests:
 	(cd build && ctest)
+	@echo "-> NGS man is available at: " && man -w ngs1
```
Comment (ilyash-b) [inline Makefile:16]: Let's not mix tests that can run after build but without installation (`ctest`) with ones that test the installation. Then, only run installation tests where appropriate.

**Review by ilyash-b — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (organom) [reply Makefile:16]: So in what place do you advice to add this tests?

Comment (ilyash-b) [reply Makefile:16]: How about new target? Let's say `test-installation`?

Comment (organom) [reply Makefile]: Added the test-installation as requested

Comment (organom) [reply Makefile]: @ilyash should we add the man testing into our actions that test ngs build ?

Comment (ilyash-b) [reply Makefile]: @organom , yes, where applicable

**Comment by organom** [comment]
Man test added to be build, after merge of this, we need to re-adapt PR #524

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -62,6 +63,13 @@ jobs:
             echo "CC=clang-${{ matrix.version }}" >> $GITHUB_ENV
             echo "CXX=clang++-${{ matrix.version }}" >> $GITHUB_ENV
           fi
+      - name: Install and Test (Linux)
+        if: runner.os == 'Linux'
+        # Sudo is required for Ubuntu 18.04 on GitHub.
+        # Otherwise "mkdir -p build/Testing/Temporary" fails with
+        # "mkdir: cannot create directory ‘build/Testing’: Permission denied"
+        # (mkdir itself was a test after "ctest" failed to create the directory)
+        run: (sudo ./install.sh && sudo make tests test-installation)
```
Comment (ilyash-b) [inline .github/workflows/build.yml:72]: Can we get rid of sudo? I think we don't have Ubuntu 18.04 anymore. If we can, it should be possible to deduplicate here.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -119,7 +128,7 @@ jobs:
       - name: Install Compiler
         run: |
           apt-get update
-          apt-get install -y sudo   # sudo is not present on the image
+          apt-get install -y man sudo   # sudo is not present on the image
```
Comment (ilyash-b) [inline .github/workflows/build.yml:131]: Why sudo is needed?

Comment (organom) [reply .github/workflows/build.yml]: legacy purposes, because the install script used to require it. Removed and build is still passing :)

**Comment by organom** [comment]
@ilyash-b changes applied, please re-review

**Pull request merged**